lib/Traits/TimeFilterTraits.ts

```diff
@@ -0,0 +1,45 @@
+import ModelTraits from "./ModelTraits";
+import LatLonHeightTraits from "./LatLonHeightTraits";
+import objectTrait from "./objectTrait";
+import primitiveTrait from "./primitiveTrait";
+import mixTraits from "./mixTraits";
+
+export class TimeFilterCoordinates extends mixTraits(LatLonHeightTraits) {
+  @primitiveTrait({
+    type: "number",
+    name: "x",
+    description: "X coordinate of the tile"
```

Comment:
These traits are very specific to how WMS works - I wonder if we should name them accordingly?

Reply:
Or perhaps make reference to WMS in the description. The focus should be on the `LatLonHeightTraits` as that's what we're filtering by, but here it's pretty easy to miss that and get confused by the `x`, `y`, `level` which are really WMS specific.

Reply:
Have renamed the traits and added a pointer to the cesium doc in the trait description.

Reply:
As per discussion with Nanda it turns out we're just following the approach of Cesium for pickFeatures